code/game/atoms.dm

```diff
@@ -1952,6 +1952,8 @@
 	return TRUE
 
 /atom/MouseEntered(location, control, params)
+	if(flags_1 & INITIALIZED_1)
+		usr.MouseEnteredOn(src, location, control, params)
```

Comment:
This is going to be bad for performance, I separated mouse enters into their own very very optimized subsystem for a reason. CC @LemonInTheDark

Reply:
We're already eating the perf hit by having on mouse entered defined at the atom level. I'm concerned that switching to.the subsystem would break ECS design and actually be slower and less responsive to players.

Reply:
We are taking *a* perf hit, but by adding an extra proc to this, especially during verb time, you are going to notice a *significant* downgrade in performance, the *whole reason* it was moved over to a subsystem



If this is for the placement silhouette, the cheapest way to do this is most likely to create a full screen UI when you are in the forklift, and use the mouse position to calculate what tile they're hovering over.

Reply:
migrated it to a signal that's off of on_mouse_enter/SSmouse_entered, still laggy as hell but gets the job done-ish

https://github.com/tgstation/tgstation/assets/4081722/b9aa9aa0-6e1b-4af1-8ac6-cc9eb69c21e4

Reply:
i actually had a bug, this is now lightning fast, fuck yes great work on the subsystem

https://github.com/tgstation/tgstation/assets/4081722/c96e8a94-b312-43b7-a516-59d1c5ede5b4